index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Homebase</title>
+  <link rel="stylesheet" type="text/css" href="./style.css">
+</head>
+
+<body>
+  <style>
+    * {
+      margin: 0px;
+      padding: 0px;
+    }
+
+    webview {
+      display: inline-flex;
+      width: 100%;
+      height: 100%;
+    }
+
+    webview.hide {
+      visibility: hidden;
+    }
+  </style>
+  <webview id="wv1" src="https://admindev.homebase.ai" allowpopups webpreferences="nativeWindowOpen=yes" autosize="on" minwidth="600"
+    minheight="800"></webview>
+  <script>
+    // You can also require other files to run in this process
+    require('./renderer.js')
```

Comment:
Can we move this to its own js file instead of inline scripting?

Reply:
Yes, the ```require('./renderer.js')``` is part of electron, though, and cannot be removed.